third_party/move/extensions/move-table-extension/src/lib.rs

```diff
@@ -42,7 +42,7 @@ use std::{
 /// The representation of a table handle. This is created from truncating a sha3-256 based
 /// hash over a transaction hash provided by the environment and a table creation counter
 /// local to the transaction.
-#[derive(Copy, Clone, Debug, PartialOrd, Ord, PartialEq, Eq)]
+#[derive(Copy, Clone, Debug, Hash, PartialOrd, Ord, PartialEq, Eq)]
```

Comment:
I think it might be better to switch to aptos_table_extension::TableHandle and derive Hash there

Reply:
leaving for separate PR